MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -102,6 +101,7 @@
 										  models[i],
 										  kPushConstDescSet,
 										  kPushConstBinding,
+										  1,
```

Comment:
Wait, I don't get it. Isn't this where you're supposed to set the variable descriptor count so the array has the right size?

Reply:
No of course not, it's the call in MVKDescriptor.mm.